### PR TITLE
Shadekin convert nutrition into energy

### PR DESCRIPTION
**Pull request by Arokha**

### CODE REVIEW

**Comment by SilverTalismen** [comment]
What is a Shadekin?

**Comment by Arokha** [comment]
`desc = "Some sort of fluffer. Big ears, long tail. "`

**Comment by SilverTalismen** [comment]
Ah okay then.

**Comment by Arokha** [comment]
(They're animalthings we use for events that can phase shift and stuff)

File: code/modules/mob/living/simple_animal/vore/shadekin/shadekin.dm [hunk 1]
```diff
@@ -191,6 +191,11 @@
 	. = ..()
 	if(ability_flags & AB_PHASE_SHIFTED)
 		density = FALSE
+
```
Comment (Leshana) [inline code/modules/mob/living/simple_animal/vore/shadekin/shadekin.dm]: I suggest they only do dis if dey be non-dead.

**Pull request merged**